bitcoin/src/taproot.rs

```diff
@@ -322,8 +322,6 @@ impl From<&TaprootSpendInfo> for TapTweakHash {
 /// See Wikipedia for more details on [DFS](https://en.wikipedia.org/wiki/Depth-first_search).
 // Similar to Taproot Builder in Bitcoin Core.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-#[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
 pub struct TaprootBuilder {
```

Comment:
Yep, IMO no builder should be (de)serializable, builders are purely Rust construct.